app/models/article.rb

```diff
@@ -1,12 +1,12 @@
-class Article
+# frozen_string_literal: true
 
+class Article
   include DataMapper::Resource
 
   property :id,           Serial
   property :title,        Text
   property :description,  Text
   property :url,          Text
 
-  belongs_to :userdigest, :required => false
-
+  belongs_to :userdigest, required: false
```

Comment:
find me to explain how we can enhance this with `belongs_to :user` and `has_many: digests`